### PR TITLE
Improved examples folder structure

### PR DESCRIPTION
**Pull request by tomaszrybakiewicz**
> DO NOT MERGE! 
> This PR must be merged **AFTER** [android-docs PR #2025](https://github.com/mapbox/android-docs/pull/2025)

Closes [#1410](https://github.com/mapbox/navigation-sdks/issues/1410) 

## Changes

- Moved all examples to individual packages.
- Updated build.gradle to register all resources folders with main source set.
- Updated CONTRIBUTING.md guide to include "Adding new example" instructions.

### CODE REVIEW

File: app/src/main/java/com/mapbox/navigation/examples/waypoints/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+<resources>
+
+  <!-- Multiple way points -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/waypoints/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/turnbyturn/res/values/strings.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Turn by Turn -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/turnbyturn/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/tripprogress/res/values/strings.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Trip progress -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/tripprogress/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/routeline/res/values/strings.xml [hunk 4]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Route line and route arrow -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/routeline/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/maneuvers/res/values/strings.xml [hunk 5]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Maneuvers -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/maneuvers/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/location/res/values/strings.xml [hunk 6]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Current Location -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/location/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/fetchroute/res/values/strings.xml [hunk 7]
```diff
@@ -0,0 +1,10 @@
+<resources>
+
+    <!-- Fetch Route -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/fetchroute/res/values/strings.xml:3]:
```suggestion
```

File: app/src/main/java/com/mapbox/navigation/examples/arrival/res/values/strings.xml [hunk 8]
```diff
@@ -0,0 +1,7 @@
+<resources>
+
+    <!-- Custom Arrival Building Extrusions -->
```
Comment (VysotskiVadim) [inline app/src/main/java/com/mapbox/navigation/examples/arrival/res/values/strings.xml:3]:
```suggestion
```

**Review by VysotskiVadim — APPROVED** [review]
I like new structure ❤️

**Comment by tomaszrybakiewicz** [comment]
Waiting for https://github.com/mapbox/android-docs/pull/2025 before merging.

**Pull request merged**